components/ai_chat/core/common/features.cc

```diff
@@ -90,4 +90,12 @@ bool IsOpenAIChatFromBraveSearchEnabled() {
   return base::FeatureList::IsEnabled(features::kOpenAIChatFromBraveSearch);
 }
 
+BASE_FEATURE(kSidebarUsePageContext,
```

Comment:
How about `kPageContextEnabledInitially` because
- This isn't disabling or enabling using page content, only the initial toggle behavior
- It's not only sidebar (on android there is no sidebar), but it's for all Conversations which start with a content association.

Reply:
fixed!